charts/db-instances/templates/postgres_exporter.yaml

```diff
@@ -89,15 +65,15 @@ metadata:
 spec:
   type: ClusterIP
   ports:
-    - port: 8080
-      targetPort: metrics
+    - port: 80
+      targetPort: http
       protocol: TCP
-      name: metrics
+      name: http
   selector:
     {{- include "db-instances.selectorLabels" $ | nindent 4 }}
     db-instance: {{ $name }}
 ---
-{{- if $.Capabilities.APIVersions.Has "monitoring.coreos.com/v1" }}
+{{- if $value.serviceMonitor.enabled }}
```

Comment:
I would like to keep using the Capabilities. I'm aware that it's not supported by ArgoCD, but we shouldn't try to deploy a resource which definition doesn't exist in the cluster. 

Maybe we can come up with something like that

```
serviceMonitor:
  enabled: true
  force: true # to disable the API check
```

And create a helper function for handling that?

Reply:
I can also leave it in. 🙂